mcu/lighting/lighting.ino

```diff
@@ -87,17 +90,30 @@ void loop() {
     hue = hue1 + place_in_span / 256;
   } else if (mode < 640) {
     // Single colour strobe
-    int spd = v[1];
-    int hue = v[2];
-    int sat = v[3];
+    int spd = map(v[1], 0, 1023, 0, 255);
```

Comment:
Do you want to scale this like below? In other words use `75` as the last value instead of `255`?